frontend/README.md

```diff
@@ -0,0 +1,16 @@
+# HTML/CSS - Interview
+
+We believe that a good front-end should be built using semantic markup and well-structured CSS. To that end, it's important that our front-end developers demonstrate the ability to convert a mocked-up page image into well-written code. The goal of this excercise is to take the mockups provided and build an HTML/CSS representation of them.
+
+## Requirements
+
+- [ ] Do not use a front-end framework such as bootstrap.
+- [ ] You can choose to either include the filler text as written, or provide your own.
+- [ ] Use the images provided in the assets directory where applicable.
+- [ ] The page should be responsive rather than have a separate mobile version.
+- [ ] Aside from the provided images, no other images should be used in the creation of your page.
+- [ ] If you do not have access to the Helvetica font, you may substitute with a sans-serif of your choosing.
+- [ ] You must use a CSS preprocessor to build your page. You may use Sass or Less.
+
+## Bonus Points
+Instead of hiding the navigation menu on mobile devices, adapt it into a mobile-friendly interface of your design. You may use JavaScript to achieve your desired results.
```

Comment:
Space between `## Bonus Points` and `Instead of ....` to be consistent with the sections above